app/src/main/java/com/gh4a/fragment/UserFragment.java

```diff
@@ -396,11 +396,13 @@ private void loadOrganizationMemberCountIfOrg(boolean force) {
         if (mUser.type() != UserType.Organization) {
             return;
         }
-        final OrganizationMemberService service =
-                ServiceFactory.get(OrganizationMemberService.class, force);
-        ApiHelpers.PageIterator
-                .toSingle(page -> service.getMembers(mUser.login(), page))
-                .map(memberList -> memberList.size())
+        final OrganizationMemberService service = ServiceFactory.get(OrganizationMemberService.class, force, 1);
+        service.getMembers(mUser.login(), 1)
+                .map(ApiHelpers::throwOnFailure)
+                .map(page -> {
+                    int pagesCount = ApiHelpers.getTotalPagesCount(page);
+                    return pagesCount == 1 ? page.items().size() : pagesCount;
```

Comment:
Is there any reason this is not just `return pagesCount;`? Since the request is 1 item per page, in what case does the current logic make a difference?

Reply:
When there are 0 members, `pagesCount` is still 1 but there are no items in the list